src/main/java/com/google/dicomwebfuse/dao/FuseDaoImpl.java

```diff
@@ -140,7 +163,27 @@ public FuseDaoImpl(AuthAdc authAdc, HttpClientFactory httpClientFactory) {
         .addParameter(PARAM_LIMIT, VALUE_PARAM_MAX_LIMIT_FOR_SERIES.toString())
         .addParameter(PARAM_OFFSET, queryBuilder.getOffset().toString())
         .setPath(path);
-    return createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {});
+    return createRequestForObjectList(uriBuilder, new TypeReference<List<Series>>() {
```

Comment:
Noop change, revert?

If you want to change the style across the entire file, please do that in a separate clean-up change (to keep diffs small and readable).

Reply:
reverted